joshuaCarpentier (copy)/assignments/slidingwindow/lc53/lc53.js

```diff
@@ -0,0 +1,67 @@
+function max_subarray_size_k(k, arr) {
```

Comment:
Would be great to see some pseudocode re: your approach and/or detailed comments.